### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader : Incorrect information is announced regarding number of rows present in table. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6151]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
JAWS version : 2019.1906.10 ILM

**Additional Details:**
Tool Name: JAWS
MAS Reference:MAS1.3.1

**Steps to Reproduce:**
1. Install the app Azure Data Studio.
2. Enable   SQL server2019 preview->select table from the database->Create External   Table screen should open->Next button->Create Database Master Key   screen should open->click on next button->fill all the required  details->click on next button.
4. Select any table/view from available options on left panel.
5. Navigate till table using up/down arrow or directly press T to jump on table.
6. Start JAWS.
7. Observe that screen reader is not announcing the incorrect information about number of rows present in the table. 

**Actual:**
Screen reader announce "grid with 5 columns and 10 rows ",but there are only 6 rows are present.
Note : By using Ctrl+alt+arrow screen reader announce first four rows as blank.

**Expected:**
Screen reader should announce correct number of rows .
for example if there are 6 rows so it should announce it like "grid with 5 columns and 6 rows".

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will get incorrect information about number of rows .

**MAS Reference:**
MAS1.3.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
Fix should be similar to https://github.com/microsoft/azuredatastudio/pull/6816

**Comment by Charles-Gagnon** [comment]
Possibly related to #6069
